### PR TITLE
Support Scala 2.13.11

### PR DESCRIPTION
**Pull request by lolgab**
After this you can release a new version

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -23,7 +23,7 @@ developers := List(
 )
 
 scalaVersion := "2.13.10"
```
Comment (saeltz) [inline build.sbt:25]:
```suggestion
scalaVersion := "2.13.11"
```

**Review by saeltz — APPROVED** [review]

**Comment by saeltz** [comment]
Thanks for your contribution!

@mccartney, would you please release a new minor version?

File: build.sbt (hunk 1 above)
Comment (lolgab) [reply build.sbt]: Using `2.13.11` as the main version fails checks for missing dependencies.

Comment (lolgab) [reply build.sbt]: I updated scoverage which should fix the problem. It seems it was using scapegoat itself creating a circular dependency but it's scoverage. :)

**Review by saeltz — APPROVED** [review]

**Comment by mccartney** [comment]
On it.

**Comment by mccartney** [comment]
2.1.2 is on its way.

**Pull request merged**